core/scripts/local/AdvanceEMP.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * This script advances time in the EMP forward by some specified amount of seconds (or one hour and one second by default).
+ *
+ * `truffle exec ./scripts/local/AdvanceEMPTime --time 10000`
```

Comment:
I think you're taking in time as seconds, so this example should be `10`.

Reply:
ah yes